package.json

```diff
@@ -42,6 +42,7 @@
     "@types/express": "^4.17.17",
     "@types/jsforce": "^1.9.33",
     "@types/node": "^18.11.11",
+    "@types/node-fetch": "^2.6.11",
```

Comment:
This PR says it's replacing the `request` library. Can we remove the dependency?